pkg/utils/v1alpha1/maya.go

```diff
@@ -100,10 +100,10 @@ func PatchCVCNodeID(volumeID, nodeID string) error {
 
 	newCVCObj, err := cvc.BuildFrom(oldCVCObj.DeepCopy()).
 		WithNodeID(nodeID).Build()
-	subresources := "publish"
```

Comment:
By doing this, aren't we patching entire object? and if any changes happens (for ex., for status) between getting the old one and doing patch for new one, that status update gets overwritten right?

Reply:
this seems to be fine as we are sending only the data bytes that need to be patched